src/main/java/com/cupid/jikting/common/repository/RedisJwtRepository.java

```diff
@@ -0,0 +1,40 @@
+package com.cupid.jikting.common.repository;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.redis.core.RedisTemplate;
+import org.springframework.data.redis.core.ValueOperations;
+import org.springframework.stereotype.Repository;
+
+import javax.annotation.PostConstruct;
+import java.time.Duration;
+
+@RequiredArgsConstructor
+@Repository
+public class RedisJwtRepository {
+
+    private final RedisTemplate<String, Object> redisTemplate;
+
+    private ValueOperations<String, Object> tokens;
+
+    @PostConstruct
+    private void init() {
+        tokens = redisTemplate.opsForValue();
+    }
+
+    public void save(String key, String value, Duration expireTime) {
+        ValueOperations<String, Object> valueOperations = tokens;
+        valueOperations.set(key, value, expireTime);
```

Comment:
p1. tokens를 다시 받아서 사용하지 않아도 됩니다!

```suggestion
        tokens.set(key, value, expireTime);
```